ui/hooks/src/use-integration-registry.ts

```diff
@@ -106,7 +106,14 @@ export function useGetIntegrationReleaseId(
 const getAllIntegrationsIds = async (offset?: number) => {
   const sdk = getSDK();
   const res = await sdk.api.icRegistry.getAllIntegrationsIds(offset);
-  return res.data;
+  let nextIndex = res.data?.nextIndex?._hex;
```

Comment:
you can use `res.data?.nextIndex?.toNumber()`

Reply:
if I cast it to number can I still pass it as offset to the sdk call?

Reply:
you can keep it as it is now `const nextRes = await sdk.api.icRegistry.getAllIntegrationsIds(res.data.nextIndex)`

Reply:
it uses the response data, not this value from `nextIndex`

Reply:
yes, only that is a bug, it should use the nextIndex

Reply:
it should work with the value casted to number

Reply:
but seems it's ok even with the nextIndex passed as a number instead of hex